src/tool_operate.c

```diff
@@ -2176,12 +2176,12 @@ static CURLcode operate_do(struct GlobalConfig *global,
 #if defined(USE_WIN32_CRYPTO) && \
    !defined(CURL_WINDOWS_OPENSSL_NEVER_AUTOMATIC_USE_WINDOWS_CA_STORE) && \
    !defined(LIBCURL_WINDOWS_OPENSSL_NEVER_USE_WINDOWS_CA_STORE)
-        if (!config->cacert && !config->capath) {
-            // now, we are under MS-Windows.
-            // user, and environement did not specify any ca file or path
-            // and there is no "curl-ca-bundle.crt" file in standard path
-            // so the only possible solution is using the windows ca store
-            config->cacert = strdup(CURL_WINDOWS_CA_STORE);            
+        if(!config->cacert && !config->capath) {
+            /* now, we are under MS-Windows.
```

Comment:
Note that this is not using a 2-space indent level, which all curl code does. You should probably make sure all your code follow that style.